src/main/openapi/ds-discover-openapi_v1.yaml

```diff
@@ -9,7 +9,7 @@ info:
     url: '${license.url}'
 servers:
   #  /api must match servlet-mapping in web.xml
-  - url: '/${project.artifactId}/v1'
+  - url: '${config.url}'
```

Comment:
I'm missing something here: How come this works when there are no `v1` in the `config.url`?

Reply:
The URL shown here is in fact equal to the one in `config.url`. When calling the endpoint: `http://localhost:9074/ds-discover/v1/ds-discover-openapi_v1.json` and examining the result you will see that the v1 part is still missing. 

I believe that the OpenAPI interface draws the version number from the `info.version` part for the header and the `servers-description`-part for the servers dropdown. 

I'll update the URL in `config.url` to include the v1.